pageStyles/constants.scss

```diff
@@ -1,19 +1,30 @@
-﻿//Default Header Width
-$welcomeHeaderWidth: 760px;
-//Site Colours
+﻿//Site Colours
 $techSwitchBlack: rgb(24, 24, 24);
 $techSwitchYellow: rgb(255, 193, 0);
 $techSwitchWhite: rgb(255, 255, 255);
 
 //Fonts
 $techSwitchFont: Open Sans, sans-serif;
-$desktopHeaderFontSize: 60px;
-$desktopSubHeaderFontSize: 36px;
-$buttonFontSize: 27px;
-$bodyFontSize: 20px;
+    //Desktop
+    $desktopHeaderFontSize: 2.5rem;
+    $desktopSubHeaderFontSize: 2rem;
+    $desktopBodyFontSize: 1.5rem;
+    
+    //Low Resolution 800px and below
+    $mediaLowResMaxWidth: 900px;
+    $lowResHeaderFontSize: 1.5rem;
+    $lowResSubHeaderFontSize: 1rem;
+    $lowResBodyFontSize: 1rem;
+
+    
+    //High Resolution 2000px and above
+    $mediaHighResMinWidth: 2500px;
+    $highResHeaderFontSize: 5rem;
+    $highResSubHeaderFontSize: 4rem;
+    $highResBodyFontSize: 3.5rem;
```

Comment:
these seem HUGE!!

I don't think they need to change very much (or perhaps at all) for high resolution displays.

Take a look at these examples
- GovUK: Doesn't change font size based on device size at all - https://design-system.service.gov.uk/styles/typography/
- BBC: Changes them slightly on small resolutions, but doesn't change once its larger than 600px wide: https://www.bbc.co.uk/gel/guidelines/typography

Reply:
Ok, will have a think about layout instead then as looks too bunched up at edges of the screen in 4K